website_blog_layout_options/models/blog_post.py

```diff
@@ -48,6 +48,21 @@ def _extract_teaser(self):
     def set_new_default(self):
         self.background_image_show = self.blog_id.background_image_show
 
+    
+    def _make_thumbnail(self):
+        import pudb
+        pudb.set_trace()
+        attachment_dict = {
+                'name': self.name + 'thumbnail',
+                'datas': self.thumbnail_binary,
+                'type': 'binary',
+                'res_model': 'ir.ui.view',
```

Comment:
this does nothing without a res_id, and probably breaks stuff because a lot of code expects the cmobination of res_id and res_model to resolve to some record. Better attach it to the blog post in question